agent/consul/fsm/commands_oss.go

```diff
@@ -562,6 +562,14 @@ func (c *FSM) applyConfigEntryOperation(buf []byte, index uint64) interface{} {
 			return err
 		}
 		return true
+	case structs.ConfigEntryUpsertStatusCAS:
+		defer metrics.MeasureSinceWithLabels([]string{"fsm", "config_entry", req.Entry.GetKind()}, time.Now(),
+			[]metrics.Label{{Name: "op", Value: "upsert_status"}})
```

Comment:
Wondering if this should be rolled in with the `"upsert"`s above for the purposes of metrics. Thoughts?

Reply:
I'm thinking we probably want to split this out just because this is more of an internal call than the others, I'm ok with changing the metric from `upsert_status` to like `internal_upsert` or `controller_upsert` or something of the sort, but I personally think you'd get more utility out of keeping the stats separate for what is user-controlled v. internal.